packages/design-system/package.json

```diff
@@ -46,9 +46,9 @@
     "jszip": "^3.10.1",
     "lodash-es": "^4.17.21",
     "p-queue": "^7.3.4",
-    "re-resizable": "^6.9.9",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "re-resizable": "^6.11.2",
```

Comment:
Let's make a list of all packages in the PR description that were required to be upgraded due to the React upgrade, so QA knows the areas to test.